server/controllers/all.js

```diff
@@ -129,8 +138,32 @@ function cleanData(world) {
     return world;
 }
 
+function encryptData(data, passphrase) {
+    let cipher = crypto.createCipher(ENCRYPTION_ALGORITHM, passphrase);
+    return Buffer.concat([
+        ENCRYPTED_CONTENT_TAG,
+        cipher.update(data),
+        cipher.final()
+    ]).toString('base64');
+}
 
-module.exports.export = async function(req, res) {
+function decryptData(data, passphrase) {
+    let rawData = new Buffer(data, 'base64');
+    let [tag, encrypted] = [rawData.slice(0, 3), rawData.slice(3)];
```

Comment:
Just tested on the CLI, you can slice a buffer as you want even if it is empty, it seems. -_-'